var/spack/repos/builtin/packages/scalasca/package.py

```diff
@@ -29,6 +29,8 @@ class Scalasca(AutotoolsPackage):
     depends_on("mpi")
 
     # version 2.4+
+    depends_on('scorep@6.0:', when='@2.4:', type=('build', 'run') )
+    depends_on('cube@4.4:', when='@2.4:', type=('build', 'run') )
```

Comment:
We don't link to these libraries? Also, are these dependencies needed for older versions?

Reply:
Adam, JSC told me they don't, the package aren't needed at build time either.